extensions/amp-analytics/amp-analytics.md

```diff
@@ -376,7 +376,11 @@ Use this configuration to fire a request when the page becomes visible. The firi
 
 ##### Visibility Spec
 Visibility spec is a set of conditions and properties that can be applied to `visible` or `hidden` triggers to change when they fire. If multiple properties are specified, they must all be true in order for a request to fire. Configuration properties supported in `visibilitySpec` are:
-  - `selector` This property can be used to specify the element to which all the `visibilitySpec` conditions apply. The selector needs to be an css id that points to an [AMP extended  element](https://github.com/ampproject/amphtml/blob/master/spec/amp-tag-addendum.md#amp-specific-tags).
+  - `selector` This property can be used to specify the element to which all the `visibilitySpec` conditions apply. The selector needs to point to an [AMP extended  element](https://github.com/ampproject/amphtml/blob/master/spec/amp-tag-addendum.md#amp-specific-tags). In addition, the semantics of how the element is selected can be changed using `selectionMethod`. The value of this property can be either a css id (without `selectionMethod` or with `selectionMethod="scope"`) or a tag name (with `selectionMethod="closest").
```

Comment:
Currently tag name with `selectionMethod = scope` is also allowed in the implementation.